libs/platform/src/lib/components/value-help-dialog/models/vhd-strategy.enum.ts

```diff
@@ -1,17 +1,3 @@
-export enum VhdDefineStrategy {
```

Comment:
@mykolavorobiov i see that you edited that file a week ago, I found an unneeded interface, removed it, but maybe it had the implicit sence, please take a look.